htdocs/bootstrap/css/custom-css.css

```diff
@@ -831,3 +831,8 @@ legend {
     font-size: 14px;
     color: #064785;
 }
+
```

Comment:
@HenriRabalais do we not already have css for modals ?

Reply:
Yes but without this change the modal does not have scroll.

Reply:
then my question becomes, is this gonna affect the modals in the biobank ??? and im pretty sure those have scrolling

Reply:
![Uploading Screenshot from 2020-10-19 16-18-16.png…]()

Reply:
@ridz1208 yeah, the modal window does have an `overflow: scroll` property.

Reply:
how come its not working for @laemtl ??

Reply:
@ridz1208 because this PR doesn't make use of the jsx/Modal.js component.

Reply:
I can take care of the rectification once the priorities have been dealt with. For now, I have integrated my style changes directly into the html to avoid conflicts with the react component.